src/NHSD.BuyingCatalogue.Ordering.OrderingDatabase/Tables/OrderStatus.sql

```diff
@@ -0,0 +1,5 @@
+﻿CREATE TABLE dbo.OrderStatus
+(
+	OrderStatusId INT NOT NULL PRIMARY KEY, 

```

Comment:
Spacing?

Reply:
I had to view this one using GitHub to understand what do you mean 😄 For some reason it looked fine in VS. 
I'll indent it properly 😅